components/landing-pages/icon-section-item.tsx

```diff
@@ -30,6 +37,26 @@ export const LandingIconSectionItem: React.FC<LandingIconSectionItemProps> = ({
           {title}
         </Text>
         <Text size="body.lg">{description}</Text>
+        {href && (
```

Comment:
let's not do all this, description already accepts a node so just pass a fragment with a tracked link at the end.